docs/build/zkEVM/index.md

```diff
@@ -6,17 +6,27 @@ import introduction from "/docs/build/img/buildZkEvm.png"
     <img src={introduction} style={{width: 1200}} />
 </div>
 
+## Overview of Monolithic versus Modular blockchains
+
+Blockchains are monolithic, by nature, and while each are able to perform four critical functions independently: **Consensus, Execution, Data Storage, and Settlement**, it's impossible for any single blockchain to perform all of them *well*. This is where more granular blockchain solutions come in. 
+
+By modularizing the blockchain protocol stack, Layer-2 solutions selectively offload functions normally provided by the Layer-1 to *highly specialized* off-chain workers designed not only to provide an equivalent, but ***superior*** experience compared to that of the underlying chain. 
+
+Modular blockchain solutions are purpose-built to *excel* at performing only the functions they are specifically intended for, such as transaction **Execution** and **Data Availability**, leaving **Settlement** to the underlying Layer-1 and in theory, advancing the challenges of blockchain scaling past those originally imposed by the blockchain trilemma.
+
 ## What is Astar zkEVM?
 
-Astar zkEVM is an Ethereum Layer-2 scaling solution leveraging Polygon's Chain Development Kit and cutting edge zero-knowledge cryptography to enable off-chain transaction execution, with finality and security guarantees provided by Ethereum. In coordination with our key partners, Astar zkEVM is well-positioned to take advantage of the extensive developer base and well-established toolset existing in the Ethereum ecosystem, and boasts the following key features:
+Astar zkEVM is a modular Layer-2 scaling solution leveraging Polygon's zero-knowledge rollups combined with Avail's validium-based storage to facilitate off-chain execution and data availability, with finality guaranteed by Ethereum. 
```

Comment:
@meganskye I don't think Avail info is something publicly communicated yet. Please verify with the team.
If it is, some external links can be added for reference.